R/derive_vars_disposition_reason.R

```diff
@@ -16,26 +23,16 @@
 #' @return A `character` vector
 #'
 #' @export
-#' @family utils_fmt
-#' @keywords utils_fmt
+#' @family deprecated
+#' @keywords deprecated
 #' @seealso [derive_vars_disposition_reason()]
-#' @examples
-#' library(dplyr, warn.conflicts = FALSE)
-#' library(admiral.test)
-#' data("admiral_dm")
-#' data("admiral_ds")
-#'
-#' # Derive DCSREAS using format_reason_default
-#' admiral_dm %>%
-#'   derive_vars_disposition_reason(
-#'     dataset_ds = admiral_ds,
-#'     new_var = DCSREAS,
-#'     reason_var = DSDECOD,
-#'     format_new_vars = format_reason_default,
-#'     filter_ds = DSCAT == "DISPOSITION EVENT"
-#'   ) %>%
-#'   select(STUDYID, USUBJID, DCSREAS)
 format_reason_default <- function(reason, reason_spe = NULL) {
+  ### DEPRECATION
+  deprecate_warn("0.10.0",
+    "format_reason_default()",
+    details = "This function is being deprecated"
```

Comment:
Common format from other deprecated functiosn for the `details` argument here is: 

"Please use `replacement_function()` instead.

Reply:
to my understanding, `format_reason_default()` is used as an input for the argument `format_new_vars` in the `derive_vars_disposition_reason()`, it doesn't have a replacement function, but rather the user will have to address the `filter_add` argument in the new function to capture the subset. So I can attach that message to the `details`